docs/podman.1.md

```diff
@@ -173,6 +173,14 @@ the exit codes follow the `chroot` standard, see below:
 
     Hooks are not used when running in rootless mode.
 
+    Podman and libpod currently support an additional `precreate` state which is called before the runtime's `create` operation.
```

Comment:
Should this be mentioned in podman-create and podman run man pages?

Reply:
> Should this be mentioned in podman-create and podman run man pages?

I don't see hooks mentioned in either of those now.  Would you just have a line in the description pointing at this man page for hook discussion?

Reply:
Yes maybe we should just point back to the podman man page.

I don't believe podman info currently lists the hooks, we should probably add information about installed hooks that could effect the way podman works to info, so when people report bugs, we could see if their are any hooks that could be screwing up.

Reply:
> Yes maybe we should just point back to the podman man page.

I've got a commit for this locally.  Once #1920 lands hook discussion in `libpod.conf(5)` as well, I can rebase this branch on top and add that commit.

> I don't believe podman info currently lists the hooks...

Is that a blocker for this PR, or can we address it in follow-up work?  And really, whether a given hook affects a given container or not depends on whether the hook's `when` block matches the container in question.  Would you just show all possible hooks?  Or just any configured hook directories?  Or something else?  I'm happy to work up something along these lines, but I'll need more details on what you're looking for.

For a given container run, you can set `--log-level=debug` to see which hooks are loaded and which of those matched, with output like:

```
time="2018-12-02T22:33:15-08:00" level=debug msg="reading hooks from /etc/containers/oci/hooks.d"
time="2018-12-02T22:33:15-08:00" level=debug msg="added hook /etc/containers/oci/hooks.d/test.json"
time="2018-12-02T22:33:15-08:00" level=debug msg="hook test.json matched; adding to stages [precreate]"
```

That seems like it covers inspectability when you have a container that's breaking and you need to figure out why.  But maybe you'd want a `--dry-run` or `--create-the-bundle-directory-and-then-exit-without-launching-the-container-from-that-bundle` or some such so folks could try to fix broken hook injection or pre-create hook execution without actually launching containers.  This feels like "future work" too me too, but I thought I'd point it out as another possible direction.